src/metabase/api/session.clj

```diff
@@ -296,7 +296,7 @@
       (api/throw-invalid-param-exception :password (tru "Invalid reset token"))))
 
 (api/defendpoint GET "/password_reset_token_valid"
-  "Check is a password reset token is valid and isn't expired."
+  "Check is a password reset token valid and isn't expired."
```

Comment:
Shouldn't it be "Check if a password reset token is valid and isn't expired"

Reply:
I think both work.
But I do like your suggestion better.